src/pages/Profile.tsx

```diff
@@ -42,49 +65,62 @@ const styles = {
       backgroundColor: "transparent",
     },
   },
-  followerCount: { fontWeight: "bold", paddingRight: 0.5 },
+  followerCount: { fontWeight: "bold" },
   followerContainer: { paddingTop: 1, display: "flex", gap: 3 },
   header: {
     alignItems: "center",
     display: "flex",
   },
   joinedDate: {
     display: "flex",
+    color: "grey",
     gap: 0.5,
     paddingTop: 1,
   },
   nameContainer: { paddingTop: 1 },
   profileContent: { padding: 2 },
+  tabs: {
+    textTransform: "none",
+  },
   tweetCount: { fontSize: 13 },
   username: {
     color: "#71797E",
-    fontSize: 16,
+    fontSize: 15,
   },
 };
 
 type ProfileContent = {
   postCount: number;
```

Comment:
Not really related, but do we need this? We can probably infer the post count by the length of the post list array. Technically it's a little risky since we're using the same store for all post lists, but in the case that we show the wrong posts we'd have bigger problems than the post count. Maybe we can make a ticket for this

Reply:
Glad you brought this up because it just reminded me of something I was thinking about earlier.

I think we'll need the postCount variable instead of inferring it by the post list array, because at some point we'll have to include pagination to be able to infinite scroll and load more data and (I think that requires another fetch). So that postCount will be the count for all-time posts, but the post list will only be like the 25 recent posts.

Another thing I forgot to do in this ticket but I think I'll squeeze into this PR is changing of this data depending on what tab you are on. Similar to how Twitter does it depending on if you are looking at replies, posts or likes.

Reply:
Yeah good point! I agree, let's leave this topic for when we tackle infinite scrolling